### DISCUSSION TITLE
q-input typescript support for type="datetime-local"

### DISCUSSION THREAD

**Discussion by mlilback** [quasarframework/quasar#13241] [Ideas / Proposals]
QInputProps.type has a limited list of valid values for the input type (text, number, email, etc.). `datetime-local` is supported by all the major browsers but is not in that list.

**Comment by diadal** [comment]
any update on this?

**Comment by yusufkandemir** [comment]
It needs to be added to here:
https://github.com/quasarframework/quasar/blob/f6a9bcf31243d1c50acfe19464f87569230edef4/ui/src/components/input/QInput.json#L27-L30
PRs are welcome
